public/app/features/dashboard-scene/edit-pane/DashboardEditPaneSplitter.tsx

```diff
@@ -46,6 +46,11 @@ export function DashboardEditPaneSplitter({ dashboard, isEditing, body, controls
     );
   }
 
+  /**
+   * Adds star button and left side actions to app chrome breadcrumb area
+   */
+  useUpdateAppChromeActions(dashboard);
```

Comment:
Linter is complaining because we return early when dd feature toggle is not enabled. Might need to add exceptions to pass ci